src/Serilog.Sinks.Datadog.Logs/Sinks/Datadog/DatadogSink.cs

```diff
@@ -27,7 +26,7 @@ public class DatadogSink : IBatchedLogEventSink, IDisposable
         private readonly IDatadogClient _client;
         private readonly Action<Exception> _exceptionHandler;
 
-#if !NETSTANDARD1_0_OR_GREATER
```

Comment:
I simply noticed that `Task.CompletedTask` is not used in .NET Framework 4.6.1 and .NET 5.0.
I guess it could be changed to only use `Task.FromResult(false)` in case of .NET 4.5.
![image](https://user-images.githubusercontent.com/17692934/210748779-5553dae5-70be-47e9-b840-b47557426655.png)

So we could change those directives like that, but this is not in scope of this PR.
![image](https://user-images.githubusercontent.com/17692934/210748757-aa5f79fc-4919-4e48-a6bc-efe234e95a41.png)

Reply:
Thanks for pointing this out :)